src/app/calendar/samplePage.tsx

```diff
@@ -0,0 +1,59 @@
+"use client";
+
+import React from "react";
+import Calendar, { CalendarEvent as CalendarEvent } from "../../components/Calendar/Calendar";
+import { styled } from "styled-components";
+
+const Centerer = styled.div`
+    display: flex;
+    justify-content: center;
+    align-items: center;
+`;
+const SizedAspectRatio = styled.div`
+    width: 100vmin;
+    height: 100vmin;
+`;
+
+const SampleCalendar: React.FC<{}> = () => {
+    const sampleEvents: CalendarEvent[] = [
+        {
+            id: "a",
+            title: "event1",
+            start: new Date("2023-07-11"), // default Date constructor assumes UTC by default
+            end: new Date("2023-07-12"),
+            backgroundColor: "red",
```

Comment:
This is just a sample page so that anyone who's using the component can have something to refer to, so I don't think we need to have them as themed colours. We could, however, add a function so that events belonging to a particular group will have a particular colour. Right now we are leaving that to whoever is using the component. Do you think that is something we should implement in the component instead? @TomCusack-Softwire